lib/ExportsStorage.js

```diff
@@ -204,8 +226,14 @@ export default class ExportsStorage {
         const promises = names.map(name =>
           this._insert({ name, pathToFile, isDefault: false, packageName }));
         promises.push(
-          ...defaultNames.map(name =>
-            this._insert({ name, pathToFile, isDefault: true, packageName })),
+          ...defaultNames.map((defaultName: string | Object) => {
+            const defaultNameObj = typeof defaultName === 'string' ? { name: defaultName, additional: false } : defaultName;
```

Comment:
This line looks a little defensive. Can we make sure that we only pass in objects instead?